pykafka/balancedconsumer.py

```diff
@@ -553,6 +566,22 @@ def _topics_changed(self, topics):
         log.debug("Rebalance triggered by topic change")
         self._rebalance()
 
+    def _zookeeper_state_changed(self, state):
+        if state in (KazooState.LOST, KazooState.SUSPENDED):
```

Comment:
It also occurs to me that there's a case to be made for bubbling up SUSPENDED states to the application layer. What you'd want to happen may be somewhat app dependent: should we continue processing messages, but not commit the offsets, or freeze consumption completely, or continue doing both? The safest thing would be to suspend all consumption completely I suppose - that would be a change from current behaviour.